packages/frontend-shared/src/gql-components/error/ErrorCodeFrame.vue

```diff
@@ -12,7 +12,7 @@
       @click="onClick"
     >
       <i-cy-document-text_x16 class="h-16px m-12px mr-8px w-16px icon-dark-indigo-500 icon-light-indigo-100" />
-      <code>{{ fileText }}</code>
+      <code class="fileText">{{ fileText }}</code>
```

Comment:
Let's replace this with `class="break-all"`. It gives us the same behavior and is a native tailwind class so we don't have to have a style tag.